src/Infrastructure.EntityFramework/Repositories/UserRepository.cs

```diff
@@ -9,16 +10,32 @@ namespace Bit.Infrastructure.EntityFramework.Repositories;
 
 public class UserRepository : Repository<Core.Entities.User, User, Guid>, IUserRepository
 {
-    public UserRepository(IServiceScopeFactory serviceScopeFactory, IMapper mapper)
+    private const string DataProtectorPurpose = "UserRepositoryProtection";
```

Comment:
⚠️ Not to overcomplicate this, but since this is used in two places we should have the constant at a place both access. It would be destructive to _ever_ change this I figure so maybe it doesn't matter, but someone else could partially change and this would be left half-broken and potentially not noticed.

Reply:
I have centralized these in our global constants file.